packages/forms/src/Components/Tabs/Tab.php

```diff
@@ -34,6 +44,14 @@ public function getColumnsConfig(): array
         return $this->columns ?? $this->getContainer()->getColumnsConfig();
     }
 
+    public function getIcon(): ?string {
+        return $this->evaluate($this->icon);
+    }
+
+    public function hasIcon(): bool {
+        return (bool) $this->getIcon();
```

Comment:
```suggestion
        return $this->icon !== null;
```

Reply:
Do we actually really need this method? `@if ($tab->getIcon())` should suffice in the Blade view.

Reply:
We use `hasX` methods elsewhere to prevent unnecessarily evaluating closures. Best to keep the pattern imo.

Reply:
We actually still need to evaluate the closure in this case, as it could return null. So `getIcon()` can be used for the check and set to `$icon`

Reply:
Just wanted to comment what @danharrin said. Also, in other places it's used like I suggested above.